playbooks/files/initiator_random_write.fio

```diff
@@ -0,0 +1,14 @@
+[write-verify]
```

Comment:
Can we move the whole `files/` directory into `sample_production`. Only the playbooks in sample_production make use of this file